Lib/test/test_dis.py

```diff
@@ -674,8 +668,25 @@ def get_disassembly(self, func, lasti=-1, wrapper=True, **kwargs):
         return output.getvalue()
 
 
-
-code_info_code_info = """\
+if sys.flags.optimize:
+    code_info_code_info = """\
+Name:              code_info
+Filename:          (.*)
+Argument count:    1
+Positional-only arguments: 0
+Kw-only arguments: 0
+Number of locals:  1
+Stack size:        3
+Flags:             OPTIMIZED, NEWLOCALS, NOFREE
+Constants:
+   0: None
+Names:
+   0: _format_code_info
+   1: _get_code_object
+Variable names:
+   0: x"""
+else:
```

Comment:
As near as I could tell, the only different between the two strings is the 1 or 2 lines of Contents, so that the two could be replaced by one f string with
```
{"   0: None\n" if sys.flags.optimize else "
 "   0: 'Formatted details of methods, functions, or code.'\n   1: None\n"}
```

Reply:
You can't put backslashes in f-strings, apparently :disappointed:

Reply:
You could use `.format(...)` instead, like we used to have to do.  Or precalculate `contents = ...` before the string.